01-24-2023-dolt-insurance-pitch.md

```diff
@@ -0,0 +1,59 @@
+---
+subject: "MySQL Database Insurance"
+date: "2023-01-24"
+ICP: "devops and/orsre managers"
+Market: "Finance, company size?, MySQL, MariaDB, etc."
+Dolt Pitch: "Dolt your backups so you fix the Atlassian problem quickly. Minutes, not days/months"
+
+---
+
+
+
+
+Email #1
+Hello,
+
+Everyone has horror stories about a database being broken with a bad query. A single mistake disrupts the functionality of your database and systems for maybe months.  
```

Comment:
```suggestion
Everyone has horror stories about a database broken by a bad query. A single mistake breaks your application for hours, days, or even [months](https://www.dolthub.com/blog/2022-04-14-atlassian-outage-prevention/).  
```

Reply:
links arent great in cold emails but we can reference this